staff/MuyZan/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,18 @@
+nav{
+    width: 20%;
+    float: left;
+    height: 100%;
+    display:inline-block;
```

Comment:
TIP is ```display: inline-block```rule required?

Reply:
The css explodes without it.
I tried to put the `display: inline` inside a main tag that includes `<nav><section><aside>` tags and delete this rule in the childs, but all crash.

Any tip, please?